cwl/tools/irobot.cwl

```diff
@@ -0,0 +1,70 @@
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: irobotclient
+
+$namespaces:
+  arv: "http://arvados.org/cwl#"   # Arvados support and api token integration 'work in progress'
+
+hints:
+
+  - class: arv:APIRequirement
```

Comment:
Ideally, both `arv:APIRequirement` and `mercury/irobot-client` should be under requirements, as they are needed for the tool to run